agent/src/main/java/org/openjdk/jmc/agent/impl/ResolvedConvertable.java

```diff
@@ -0,0 +1,54 @@
+package org.openjdk.jmc.agent.impl;
+
+import java.lang.reflect.Method;
+
+import org.openjdk.jmc.agent.Agent;
+import org.openjdk.jmc.agent.Convertable;
+
+public class ResolvedConvertable extends AbstractConvertable implements Convertable {
+	private final static String CONVERTER_METHOD = "convert";
+	private final transient Class<?> converterClass;
```

Comment:
When is this ResolvedConvertable serialized, i.e. why is this transient?

Reply:
Ah, was from an earlier version of the code where I let the attributes be able to resolve themselves. Removed.